client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -69,10 +70,19 @@ export default React.createClass( {
 		analytics.pageView.record( '/checkout/domain-contact-information', 'Checkout > Domain Contact Information' );
 	},
 
-	validate( fieldNames, onComplete ) {
+	trimWhitespace( fieldValues, onComplete ) {
+		const trimmedFieldValues = Object.assign( {}, fieldValues );
+		this.fieldNames.forEach( ( fieldName ) => {
+			trimmedFieldValues[ fieldName ] = fieldValues[ fieldName ].trim();
+		} );
+
+		onComplete( trimmedFieldValues );
+	},
+
+	validate( fieldValues, onComplete ) {
```

Comment:
Just changed the variable's name from `fieldNames` to `fieldValues` to mirror the names in `trimWhitespace` and, just as importantly, to avoid confusion with `this.fieldNames` :)